data/address/index.html

```diff
@@ -31,19 +31,25 @@
     <td>Text that describes the address coordinate systems in Utah.</td>
   </tr>
   <tr>
-    <td><a href="https://api.mapserv.utah.gov">UGRC Web Api for Geocoding</a></td>
+    <td><a href="{% link data/address-geocoders-locators/index.html %}#WebAPI">UGRC API for Geocoding</a></td>
     <td>Geocoding API</td>
     <td>UGRC</td>
-    <td>Custom geocoding http web api</td>
+    <td>The foundational service for all Utah geocoding and query access to the SGID.</td>
   </tr>
   <tr>
-    <td><a href="{% link data/address-geocoders-locators/index.html %}#GeocodingToolbox">Utah Geocoding Toolbox</a></td>
+    <td><a href="{% link data/address-geocoders-locators/index.html %}#OfficialClient">Official UGRC API Client for Geocoding</a></td>
+    <td>Cross-Platform Desktop Geocoding Application</td>
+    <td>UGRC</td>
+    <td>The recommended client and easiest application to use for geocoding Utah addresses.</td>
```

Comment:
easiest?  I suppose.  Just thinking if you're going to geocode an address in Pro, this wouldn't be the easiest.  but, i know what you're saying. it's the most direct and simple way to geocode a table of addresses.

Reply:
You think installing pro, downloading a toolbox, adding the toolbox, and using it to geocode is simpler?

Reply:
TBH, I agree with Steve on this one. I've fully transitioned to using the API client for geocoding. It just feels slicker/smoother/easier. 

I suppose the one thing you lose is the guided workflow in Pro for reviewing matches and fixing unmatched points, but that's easy enough.

Reply:
mm, I haven't tried using masquerade for geocoding yet.

Reply:
yes, i was going with the idea that as scott said, "if you are already working in Pro, masquerade would be easier than downloading the API client"